src/System.Windows.Extensions/tests/System.Windows.Extensions.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{AC1A1515-70D8-42E4-9B19-A72F739E974C}</ProjectGuid>
-    <Configurations>netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;netfx-Windows_NT-Debug;netfx-Windows_NT-Release</Configurations>
+    <Configurations>netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;</Configurations>
```

Comment:
@ericstj is this here a low enough risk change at this point?

Reply:
I asked @Anipik to do this change as it was a oversight from the netfx test configuration removal. We don't produce a netfx asset for System.Windows.Extensions therefore I tend towards removing the configuration unless we need the distinction in behavior differences.

Reply:
Regarding your question, there is no risk involved in this change.

Reply:
Well, I don't think any code change is "no risk" but my concern was we were changing the packaging, and it sounds like we're not so OK.

Reply:
Oh, of course, it's tests not product. my bad.